pkgs/development/compilers/llvm/common/openmp/default.nix

```diff
@@ -61,6 +69,10 @@ stdenv.mkDerivation (rec {
     "-DCLANG_TOOL=${clang-unwrapped}/bin/clang"
     "-DOPT_TOOL=${llvm}/bin/opt"
     "-DLINK_TOOL=${llvm}/bin/llvm-link"
+  ] ++ [
+    (lib.cmakeBool "LIBOMP_ENABLE_SHARED" (!stdenv.hostPlatform.isStatic && stdenv.hostPlatform.hasSharedLibraries))
+    (lib.cmakeBool "LIBOMP_OMPD_SUPPORT" ompdSupport)
+    (lib.cmakeBool "LIBOMP_OMPD_GDB_SUPPORT" ompdGdbSupport)
```

Comment:
Usually we'd put flags that aren't behind an `optionals` like these ones before the optional ones.

Reply:
Done.